docs/source/feature_finder_metabo_ident.rst

```diff
@@ -0,0 +1,106 @@
+FeatureFinderMetaboIdent
+========================
+
+This tool detects quantitative features in MS1 data for a list of targets specified in an assay library (a tab-separated text file). 
```

Comment:
maye rephrase and only talk about the algorithm.
E.g.,
FeatureFinderAlgorithmMetaboIdent performs MS1-based targeted feature extraction based on user provided compounds. ...